vaadin-checkbox-flow-parent/vaadin-checkbox-flow-integration-tests/src/test/java/com/vaadin/flow/component/checkbox/tests/HelperIT.java

```diff
@@ -15,27 +15,86 @@
  */
 package com.vaadin.flow.component.checkbox.tests;
 
+import com.vaadin.flow.component.checkbox.testbench.CheckboxGroupElement;
+import com.vaadin.flow.component.checkbox.testbench.CheckboxElement;
 import com.vaadin.flow.testutil.TestPath;
 import com.vaadin.testbench.TestBenchElement;
 import com.vaadin.tests.AbstractComponentIT;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
+import java.util.List;
+
 @TestPath("vaadin-checkbox/helper")
 public class HelperIT extends AbstractComponentIT {
 
+    @Before
+    public void init() {
+        open();
+    }
+
     /**
      * Assert that helper component exists after setItems.
      * https://github.com/vaadin/vaadin-checkbox/issues/191
      */
     @Test
     public void assertHelperComponentExists() {
```

Comment:
```suggestion
    public void checkboxGroupHelperComponent() {
```

IMO `assert` usually indicates a helper method for doing assertions, rather than being an actual test method. Looks we can easily refactor the naming of the tests in this case.